### DISCUSSION TITLE
Saving the Contents of Wizard and Continuing Later

### DISCUSSION THREAD

**Discussion by ssmusoke** [spatie/laravel-livewire-wizard#90] [General]
I would like to save the state of the wizard and continue later

I am thinking:

1. I am thinking I could save some information on a step in the database then load it with the wizard however I am not clear how to dynamically add the current component
2. Is there an easy way to mount the latest step dynamically? I know there is https://spatie.be/docs/laravel-livewire-wizard/v1/usage/navigating-steps#content-start-at-a-specific-step but wondering how to do it dynamically

**Comment by ju5t** [comment]
Something like this would work.

```blade
<livewire:checkout-wizard :show-step="$stepName" :initial-state="$initialState" />
```

You can set `$stepName` and `$initialState` yourself based on what's in the database (or session).

**Reply by ssmusoke** [reply]
This totally solves my need, thank you very much!!!
